web/src/narrow_banner.js

```diff
@@ -283,7 +283,7 @@ function pick_empty_narrow_banner() {
         case "search": {
             // You are narrowed to empty search results.
             return {
-                title: $t({defaultMessage: "No search results."}),
+                title: $t({defaultMessage: "No search results"}),
```

Comment:
I don't think this string change is an intentional design change; Vlad's designs generally don't aim to perfectly transcribe strings.